backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/PokerController.java

```diff
@@ -0,0 +1,106 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.web.bind.annotation.*;
+
+import static com.cgi.retrospecto.backend.poker.api.PokerConstants.ApiConstants.*;
+
+@RestController
+@RequestMapping(POKER)
+@CrossOrigin
+public class PokerController {
+    @Autowired
+    private RoomService roomService;
+
+    @ResponseBody
+    @RequestMapping(path = ROOM, method = RequestMethod.POST, consumes = MediaType.TEXT_PLAIN_VALUE)
+    public ResponseEntity<RoomDetails> createRoom(@RequestBody String author) {
+        return new ResponseEntity<>(
+                RoomConverter.toDTO(
+                        roomService.createRoom(
+                                RoomConverter.toEntity(author)
+                        )
+                ), HttpStatus.OK);
+    }
+
+    @ResponseBody
+    @RequestMapping(path = GET_ROOM, method = RequestMethod.GET)
+    public ResponseEntity<RoomDetails> getRoom(
+            @PathVariable(PokerConstants.ID) int id,
+            @PathVariable(PokerConstants.USERNAME) String username)
+            throws RoomNotFoundException {
+        Room room = roomService.getPokerRoom(id);
+        room.addUser(username);
+        return new ResponseEntity<>(RoomConverter.toDTO(room), HttpStatus.OK);
+    }
+
+    @MessageMapping(POKER + "/{roomId}/story/add")
+    @SendTo("/topic" + POKER + "/{roomId}/story/add")
+    public ResponseEntity<Story> createStory(
+            @DestinationVariable int roomId,
+            @Payload String story) throws RoomNotFoundException {
+        System.out.println("Creating story: " + story + " --> " + roomId);
+        return new ResponseEntity<>(
+                roomService.createStory(roomId,
+                        StoryConverter.toEntity(story)
+                ), HttpStatus.OK);
+    }
+
+    @MessageMapping(POKER + "/{roomId}/story/{storyId}/vote")
+    @SendTo("/topic" + POKER + "/{roomId}/vote")
+    public ResponseEntity<RoomDetails> vote(
+            @DestinationVariable int roomId,
+            @DestinationVariable int storyId,
+            @Payload Vote dto) throws RoomNotFoundException, StoryNotFoundException {
+        roomService.vote(roomId, storyId, dto);
+        return new ResponseEntity<>(
+                RoomConverter.toDTO(
+                        roomService.getPokerRoom(roomId)
+                ), HttpStatus.OK);
+    }
+
+    @MessageMapping(POKER + "/{roomId}/story/{storyId}/selected")
+    @SendTo("/topic" + POKER + "/{roomId}/selectedStory")
+    public ResponseEntity<SelectedStoryDetails> selectStory(
```

Comment:
I think if we redesign the frontend we can skip this action, basically the Scrum Master would already create a list of stories before the meeting and then it would automatically step forward in the list once a story was voted on.